{{ cookiecutter.package_name }}/setup.py

```diff
@@ -4,14 +4,10 @@
 import glob
 import os
 import sys
-{% if cookiecutter.minimum_python_version.startswith("2") %}
 try:
     from configparser import ConfigParser
 except ImportError:
     from ConfigParser import ConfigParser
-{%- else %}
```

Comment:
Probably should add a comment about this otherwise someone will add this back again in future

Reply:
damn it, thought I did, must have undone it by mistake.